smoke-test.sh

```diff
@@ -15,11 +15,18 @@ host_with_port=$(echo "$url" | sed 's/\/.*$//')
 host=$(echo "$host_with_port" | sed 's/:.*$//')
 port=$(echo "$host_with_port" | grep : | sed -e 's,^.*:,:,g' -e 's,.*:\([0-9]*\).*,\1,g')
 
-os::log::info "Getting certificate for ${CONSOLE_URL}"
-timeout 30 google-chrome --no-sandbox --headless --disable-gpu --dump-dom "$CONSOLE_URL" &>/dev/null
-echo 'GET /' | timeout 30 openssl s_client -showcerts -connect "$host:$port" | openssl x509 -outform PEM > /tmp/console-e2e.pem
-os::log::info "Storing obtained certificate"
-certutil -d "sql:$HOME/.pki/nssdb" -A -n console -t Pu,, -i /tmp/console-e2e.pem
+if [ "$SELF_SIGN_CERT" = true ]; then
+  os::log::info "Getting certificate for ${CONSOLE_URL}"
```

Comment:
Shall we update the log to "Getting self signed certificate...." just to help ensure this is super clear?